src/init/features/hosting/github.ts

```diff
@@ -392,7 +392,7 @@ async function promptForRepo(
       type: "input",
       name: "repo",
       default: defaultGithubRepo(), // TODO look at github origin
-      message: "For which GitHub repository would you like to set up a GitHub workflow?",
+      message: "For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository)",
```

Comment:
There's some lint failure on this line.

Reply:
I'm staring at the line and I've no clue which linting problem could it be... the only idea I can come with is that CLI message have length constraints but, that doesn't make much sense...
Anyway, not a big deal the contribution. We can drop it :(